From: SeongJae Park <sj@kernel.org>
Date: Wed, 19 Nov 2025 20:57:35 -0800
Subject: [PATCH] mm/damon/core: rename damos_filter_out() to
 damos_core_filter_out()

It is easy to be confused that it is only for core layer filters.  Make
name be more explicit.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index 316ce3c3728d..5b82ab1f9e77 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -2040,7 +2040,7 @@ static bool damos_filter_match(struct damon_ctx *ctx, struct damon_target *t,
 	return matched == filter->matching;
 }
 
-static bool damos_filter_out(struct damon_ctx *ctx, struct damon_target *t,
+static bool damos_core_filter_out(struct damon_ctx *ctx, struct damon_target *t,
 		struct damon_region *r, struct damos *s)
 {
 	struct damos_filter *filter;
@@ -2196,7 +2196,7 @@ static void damos_apply_scheme(struct damon_ctx *c, struct damon_target *t,
 			}
 			damon_split_region_at(t, r, sz);
 		}
-		if (damos_filter_out(c, t, r, s))
+		if (damos_core_filter_out(c, t, r, s))
 			return;
 		ktime_get_coarse_ts64(&begin);
 		trace_damos_before_apply(cidx, sidx, tidx, r,
-- 
2.47.3

